internal/core/ports/identity_service.go

```diff
@@ -39,7 +39,7 @@ type IdentityService interface {
 	UpdateIdentityState(ctx context.Context, state *domain.IdentityState) error
 	GetTransactedStates(ctx context.Context) ([]domain.IdentityState, error)
 	GetStates(ctx context.Context, issuerDID w3c.DID) ([]domain.IdentityState, error)
-	CreateAuthenticationQRCode(ctx context.Context, serverURL string, issuerDID w3c.DID) (string, error)
+	CreateAuthenticationQRCode(ctx context.Context, serverURL string, issuerDID w3c.DID) (string, string, error)
```

Comment:
```suggestion
	CreateAuthenticationQRCode(ctx context.Context, serverURL string, issuerDID w3c.DID) (string, uuid.UUID, error)
```
Returning uuid gives more info to the caller.

Reply:
sure, done!